docs/dev/build.md

```diff
@@ -24,11 +24,14 @@ For the OpenVINO build next tools are required:
 <details><summary>Windows</summary>
 <p>
```

Comment:
I think we can add a link to static OpenVINO build

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2